### PR TITLE
`Trusted Entitlements`: Add `verification` field to EntitlementInfo and EntitlementInfos

### PR DESCRIPTION
**Pull request by tonidero**
This adds `verification` to `EntitlementInfo` and `EntitlementInfos`. This will be a string version of the `VerificationResult` enum.

### CODE REVIEW

File: ios/PurchasesHybridCommon/PurchasesHybridCommonTests/EntitlementInfo+HybridAdditionsTests.swift [hunk 1]
```diff
@@ -70,14 +79,4 @@ class EntitlementInfoHybridAdditionsTests: QuickSpec {
             requestDate: .init()
         )
     }
-
-    private func mockProductData(withStore store: String) -> [String: Any] {
```
Comment (tonidero) [inline ios/PurchasesHybridCommon/PurchasesHybridCommonTests/EntitlementInfo+HybridAdditionsTests.swift:74]: This was unused

File: typescript/apitesters/enums.ts [hunk 2]
```diff
@@ -5,9 +5,10 @@ import {
   REFUND_REQUEST_STATUS,
   INTRO_ELIGIBILITY_STATUS,
   PRORATION_MODE,
-  IN_APP_MESSAGE_TYPE
+  IN_APP_MESSAGE_TYPE,
+  ENTITLEMENT_VERIFICATION_MODE,
+  VERIFICATION_RESULT
 } from '../dist';
-import { ENTITLEMENT_VERIFICATION_MODE } from '../src';
```
Comment (vegaro) [inline typescript/apitesters/enums.ts:10]: 👍

File: typescript/src/enums.ts [hunk 3]
```diff
@@ -119,3 +119,37 @@ export enum ENTITLEMENT_VERIFICATION_MODE {
   // Add ENFORCED mode once we're ready to ship it.
   // ENFORCED = "ENFORCED"
 }
+
+/**
+ * The result of the verification process.
```
Comment (NachoSoto) [inline typescript/src/enums.ts:124]: Maybe add a link to the docs here too?

http://rev.cat/trusted-entitlements

**Review by NachoSoto — APPROVED** [review]
👏🏻

**Review by vegaro — APPROVED** [review]
Looks good!

**Pull request merged**